packaging/debian-sid/snapd.postrm

```diff
@@ -126,6 +126,12 @@ if [ "$1" = "purge" ]; then
         umount -l /run/snapd/ns/ || true
     fi
 
+    # inside containers we have a generator that creates a bind mount to /snap
+    if [ -e /run/systemd/container ]; then
+        echo "Unmount /snap inside a container"
+        umount /snap || true
```

Comment:
I think I would prefer a `systemctl stop` instead but perhaps I'm mistaken and this is actually the right way to do this.